src/ome_zarr_models/_rfc5_transforms/image.py

```diff
@@ -0,0 +1,24 @@
+from pydantic import Field
+
+from ome_zarr_models._rfc5_transforms.base import BaseGroupv05, BaseOMEAttrs
+from ome_zarr_models._rfc5_transforms.multiscales import Multiscale
+
+__all__ = ["Image", "ImageAttrs"]
+
+
+class ImageAttrs(BaseOMEAttrs):
+    """
+    Model for the metadata of OME-Zarr data.
+    """
+
+    multiscales: list[Multiscale] = Field(
```

Comment:
there's an RFC that makes multiscales singular. Not sure if that goes before or after the transformations though.

Reply:
True, maybe easier to implement that RFC before.

Reply:
Yeah, it's https://github.com/ome/ngff/pull/285, but I don't have much hope for it going anywhere soon... so I think lets leave this as is. Easy enough to change it later.